userspace/libsinsp/cri.cpp

```diff
@@ -153,7 +153,22 @@ bool walk_down_json(const Json::Value &root, const Json::Value **out, const std:
 	return false;
 }
 
-bool set_numeric(const Json::Value &dict, const std::string &key, int64_t &val)
+bool set_numeric_32(const Json::Value& dict, const std::string& key, int32_t& val)
```

Comment:
I'd probably just make the cpuset count an int64 :)

Reply:
Fair but this is in the docker engine already so I'll stay consistent.